src/containers/auth/ImportScreen.tsx

```diff
@@ -3,7 +3,12 @@ import { inject, observer } from 'mobx-react';
 import { StoresProps } from '../../@types/ferdium-components.types';
 import Import from '../../components/auth/Import';
 
-class ImportScreen extends Component<StoresProps> {
+interface IProps extends StoresProps {}
+
+//  TODO - [TECH DEBT][NOT TESTED][PROPER UI NAVIGATION NOT FOUND] Need to verify if this page is still working.
```

Comment:
when you are logged out of Ferdium, there will be a screen where you can import settings.

Reply:
ok will test this screen and update here

Reply:
is it some automatic logout?
I have tried manual logout after feridium account login, it redirect me to initial screen i.e

![image](https://user-images.githubusercontent.com/104364298/201451272-8e51405e-9894-4f56-a211-40ca37f378b4.png)

Reply:
after logging out, go to "use without server", and then there should be something to import.

Reply:
is there a way to clear app data ?
as I have tried both login and use without server they seems to be loading separate sessions
intermediate screen for import is not loading

Reply:
other than wiping out your whole profile folder, i dont know any other mechanism

Reply:
how to do that in debug mode let me try it there first?

Reply:
`rm -rf ~/Library/Application\ Support/FerdiumDev/`

Reply:
Marking PR as draft till I confirm the screen.
It didn't worked out. Still seeing start screen same as above after login and logout even after using the command.
am Im missing something ?
tried app with or without rm -rf ~/Library/Application\ Support/FerdiumDev/ and clean build(packaged) , debug

Reply:
ok - im out of ideas. If someone can figure out where this screen comes, we can hold this PR. If not, lets time-box the hold, and merge this in - say in 48 hrs?

Reply:
ok sure

Reply:
As 48 hours over , I've rebased the pr and marked ready for merged

Reply:
I did try to get to that page which, according to `route.tsx`, is located at `/auth/signup/import` by tricking one of the redirection from the welcome page, but it didn't show me anything and I haven't seen it anywhere either in accountless mode, or by creating an account (could it be only for custom servers?).

Reply:
If its dead code, please go ahead and remove it in this PR itself.

Reply:
ok i shall remove !

Reply:
@vraravam removed import and its component!